### ISSUE TITLE
[Accessibility]: In Code Monitors create/edit page, errors when calling webhooks/sending email should be announced

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#35732]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34195

### Problem description

When calling a test webhook/email, if there is an error, the error is not announced.

### Expected behavior

The error shown when a config value is missing should *not* be announced (should still be visible to screen reader when focused, etc).
The error shown after trying to call the webhook/send an email and getting an error should be announced, using aria-live or similar.

![CleanShot 2022-05-19 at 11 20 34](https://user-images.githubusercontent.com/206864/169372675-6e0d15a2-7a0c-4571-a28f-b27dc285c052.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
@limitedmage The root issue we are facing comes from the `<Alert />` usages in `SlackWebhookAction` and `WebhookAction`, the `Alert` has `role=alert` + `aria-live` so browser decides to read it first and ignore others `aria-live`, we can fix the issue with binding `aria-live=off` into `Alert` usages on those component but the `Alert` content wouldn't be read. WDYT about this?

cc @umpox @valerybugakov

**Comment by umpox** [comment]
> we can fix the issue with binding aria-live=off into Alert usages on those component

This seems like a good solution and is what we (and MDN) recommends for these use cases. See [here](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/wildcard/src/components/Alert/Alert.tsx?L26%3A1-27%3A1=)
